src/plugins/embeddable/public/lib/test_samples/embeddables/filterable_embeddable_factory.ts

```diff
@@ -23,10 +23,11 @@ import {
   FilterableEmbeddableInput,
   FILTERABLE_EMBEDDABLE,
 } from './filterable_embeddable';
-import { EmbeddableFactory } from '../../embeddables';
+import { EmbeddableFactoryDefinition } from '../../embeddables';
 import { IContainer } from '../../containers';
 
-export class FilterableEmbeddableFactory extends EmbeddableFactory<FilterableEmbeddableInput> {
+export class FilterableEmbeddableFactory
+  implements EmbeddableFactoryDefinition<FilterableEmbeddableInput> {
```

Comment:
In this class below `isEditable`, `getDisplayName` and `create` should all be class properties, according to `defaultEmbeddableFactoryProvider` implementation.

```ts
idEditable: () => ..
getDisplayName: () => ..
create: () => ..
```

Reply:
I guess the same applies to other factories, too.

Reply:
I added the binding, so I think this no longer matters?  Or is it still more correct to switch them over?

Reply:
If you have `.bind` then switching to `() => ...` should not be needed.